Documentation/PageTsconfig/Mod/WebTs.rst

```diff
@@ -1,48 +1,46 @@
-.. include:: /Includes.rst.txt
+:orphan:
+..  include:: /Includes.rst.txt
 
 ======
 web_ts
 ======
 
-Configuration options of the "Web > Template" module.
+..  versionchanged:: 12.0
+    The :guilabel:`Web > Template` module has been replaced by the
+    :guilabel:`Web > TypoScript` module. The only option in this namespace
+    has been removed.
 
-.. contents::
-   :local:
+..  contents::
+    :local:
 
-.. index::
-   web_info.menu.function
-   Module menu; Template
-.. _pageblindingfunctionmenuoptions-webts:
+..  index::
+    web_info.menu.function
+    Module menu; Template
+..  _pageblindingfunctionmenuoptions-webts:
 
 menu.function
 =============
 
-:aspect:`Datatype`
-   array
+..  versionchanged:: 12.0
+    The TSconfig option :tsconfig:`mod.web_ts.menu.function` has been removed
+    with TYPO3 v12.0.
 
-:aspect:`Description`
-   Disable elements of the "Function selector" in the document header of the module. The keys for single
-   items can be found by browsing *System > Configuration > $GLOBALS['TBE_MODULES_EXT']*.
+..  _pageblindingfunctionmenuoptions-webts-migration:
 
-   .. figure:: /Images/ManualScreenshots/Template/FunctionMenuTemplateModule.png
-      :alt: The function menu of the Template module
+Migration from menu.function to options.hideModules
+---------------------------------------------------
 
-      The function menu of the Template module
+..  todo: link to options.hideModules once it is documented
```

Comment:
hideModules is documented, it is user TSconfig:
https://docs.typo3.org/m/typo3/reference-tsconfig/12.4/en-us/UserTsconfig/Options.html#hidemodules